include/net/downloader.h

```diff
@@ -195,6 +196,11 @@ struct downloader_host_cfg {
 	 * This string is used in case you are requesting a proxied file from a CoAP server.
 	 */
 	const char *proxy_uri;
+	/**
+	 * Maximum number of HTTP redirects.
+	 * Use 0 to set default value of CONFIG_DOWNLOADER_MAX_REDIRECTS.
+	 */
+	int redirects_max;
```

Comment:
So, there is no way to disable this feature by setting the maximum number of redirects to zero?